SDK Showcase/SDK Showcase/Views/ContentView.swift

```diff
@@ -58,6 +59,7 @@ struct ContentView: View {
                     Section(content: {
                         TextResult(result: system.isSDKInitialized ? "✅ SDK initialized" : "❌ SDK not initialized \(errorValue)")
                         TextResult(result: system.registationState == .registered ? "👤 User registered as \(userData.userId ?? "")" : "🚫 User not registered")
+                        TextResult(result: system.authenticationState == .authenticated ? "👤 User authenticated as \(userData.userId ?? "")" : "🚫 User not authenticated")
```

Comment:
The result should be shared between registered and authenticated (transparent to the user)
No separate row here should be shown for user

Reply:
not addressed

Reply:
Done.